pytest_inmanta/test_parameter/optional_boolean_parameter.py

```diff
@@ -0,0 +1,57 @@
+"""
+    Copyright 2023 Inmanta
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+    Contact: code@inmanta.com
+"""
+import argparse
+from typing import Optional, Type, Union
+
+from pytest_inmanta.test_parameter.boolean_parameter import BooleanTestParameter
+from pytest_inmanta.test_parameter.parameter import DynamicDefault
+
+
+class OptionalBooleanTestParameter(BooleanTestParameter):
```

Comment:
I have a tiny preference to put this in the same module as the normal `BooleanTestParameter`. I think if I were searching for this class, that's where I'd look (grep aside).

Reply:
They are currently one-file-per-class, so I kept to that (even if I would put all of then in one file)

Reply:
If you want, I'll move it.